src/material-examples/cdk-popover-edit-cdk-table/cdk-popover-edit-cdk-table-example.css

```diff
@@ -0,0 +1,7 @@
+table {
```

Comment:
nit: we always use CSS classes prefixed with `.example-` for examples. We do this because:
* It doesn't encourage people to write overly broad CSS rules on their own
* It makes it clear that it's an example style
* We turn off view encapsulation pretty frequently

(here and elsewhere)